pkg/ddevapp/global_dotddev_assets/commands/host/mailpit

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+## #ddev-generated: If you want to edit and own this file, remove this line.
+## Description: Launch a browser with Mailpit
```

Comment:
Let's make this a bit better, explaining what mailpit is? "Mail capture UI 'mailpit'" ?

Reply:
Sure thing.
https://mailpit.axllent.org/ says mailpit is an "email & SMTP testing tool" - I'll use their own wording here.